### PR TITLE
add scope permissions for media files

### PR DESCRIPTION
**Pull request by annashamray**
**Changes:**
1. Add scope permission for retrieving uploaded files

### CODE REVIEW

File: src/drc/api/permissions.py [hunk 1]
```diff
@@ -18,3 +20,13 @@ class InformationObjectRelatedAuthScopesRequired(RelatedObjAuthScopesRequired):
     """
     permission_fields = ('informatieobjecttype', 'vertrouwelijkheidaanduiding')
     obj_path = 'informatieobject'
+
+
+def allow_scopes(private_file) -> bool:
```
Comment (sergei-maertens) [inline src/drc/api/permissions.py]: This is unused and can be removed :)

Comment (annashamray) [reply src/drc/api/permissions.py]: function is removed :)

**Comment by sergei-maertens** [comment]
@annashamray I'll leave this pending for now, since I need to tackle the Docker-side of things as well and don't want to break downloads _for now_. Hoping to merge this next week!

**Comment by sergei-maertens** [comment]
test this please

**Pull request merged**